custom-routes-vis/src/main/webapp/script.js

```diff
@@ -182,11 +186,11 @@ function generateRoutes() {
   let apiKey = document.getElementById('api-key').value;
 
   // Get the routes and display them on map.
-  fetch('/get-directions?origin=' + origin + '&destination=' + destination
-      + '&endpoint=' + serviceEndpoint + '&apiKey=' + apiKey)
+  fetch('/get-directions?origin=' + origin + '&destination=' + destination + 
+      '&endpoint=' + serviceEndpoint + '&apiKey=' + apiKey)
       .then(response => response.json()).then(directions => {
         // Log the response status from the Directions API.
-        // TODO(chenyuz): find equivalence for Routes Preferred API.
+        // TODO(chenyuz): Find equivalence for Routes Preferred API.
         console.log(directions.status);
```

Comment:
If you will eventually delete this, please add a TODO to delete it.

Reply:
I'm not gonna delete the console log because it logs whether the retrieval is successful